guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+The plugin consists of three components: services running on {ProjectServer} and {SmartProxyServer}, and the Discovery image running on host.
+
+[[Discovery_Rule]]
+Discovery Rule:: A set of predefined provisioning rules which assigns a host group to discovered hosts and triggers provisioning automatically.
+
+[[Docker_Tag]]
+Docker Tag:: A mark used to differentiate container images, typically by the version of the application stored in the image.
+In the {ProjectWebUI}, you can filter images by tag under *Content* > *Docker Tags*.
+
+ifndef::satellite[]
+[[Enterprise_Linux]]
+Enterprise Linux:: An umbrella term for the following {RHEL}-like operating systems:
+
+ AlmaLinux
+ CentOS Linux
+ CentOS Stream
+ Oracle Linux
+ {RHEL}
+ Rocky Linux
+
+ifndef::orcharhino[]
+{Project} is tested on AlmaLinux and CentOS Stream.
+endif::[]
+endif::[]
+
+[[ERB]]
+ERB:: Embedded Ruby (ERB) is a template syntax used in provisioning and job templates.
+
+[[Errata]]
+Errata:: Updated RPM packages containing security fixes, bug fixes, and enhancements.
+In relationship to a host, erratum is *applicable* if it updates a package installed on the host and *installable* if it is present in the host's content view (which means it is accessible for installation on the host).
+
+[[External_Node_Classifier]]
+External Node Classifier:: A construct that provides additional data for a server to use when configuring hosts.
+{ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
++
+Note that the External Node Classifier will be removed in the next {Project} version.
+
+[[Facter]]
+Facter:: A program that provides information (facts) about the system on which it is run; for example, Facter can report total memory, operating system version, architecture, and more.
+Puppet modules enable specific configurations based on host data gathered by Facter.
+
+[[Facts]]
+Facts:: Host parameters such as total memory, operating system version, or architecture.
+Facts are reported by Facter and used by Puppet.
+
+[[Foreman]]
+Foreman:: The component mainly responsible for provisioning and content lifecycle management.
+ifdef::satellite[]
+Foreman is the main upstream counterpart of {ProjectName}.
+endif::[]
+
+[[{project-context}_Services]]
+{Project} services:: A set of services that {ProjectServer} and {SmartProxyServers} use for operation.
+You can use the `{foreman-maintain}` tool to manage these services.
+To see the full list of services, enter the `{foreman-maintain} service list` command on the machine where {Project} or {SmartProxyServer} is installed.
+
+[[Foreman_Hook]]
+Foreman Hook:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
```

Comment:
IMO this should either be a proper note (`[NOTE]`), or a plain sentence (`Foreman Hook functionality [...]`).

Reply:
I missed the part about "next version" completely. Great catch, thanks! Yes, I added a suggestion to rephrase that along with another similar occurrence.